gradle/spotbugs-exclude.xml

```diff
@@ -272,6 +272,14 @@ For a detailed description of spotbugs bug categories, see https://spotbugs.read
         <Bug pattern="IS2_INCONSISTENT_SYNC" />
     </Match>
 
+    <Match>
+        <!-- Suppresses inconsistent synchronization errors on subscription and subscribedPattern fields.
```

Comment:
The warnings all seem to be due to the use of the `Predicate` in `assignFromSubscribed`. Do you think it would be a good idea to rewrite that as a boring for loop so that we could get rid of this warning?